### PR TITLE
Remove selectbox for non-admin members(#6516)

### PR DESCRIPTION
**Pull request by sjkadali**
#6516

@paulbert , @lmmrssa  please review the PR. Thanks!

### CODE REVIEW

**Comment by sjkadali** [comment]
@paulbert thanks for clarifying the issue. When I added only the check for isDialog and  on logging in as Admin, select box was was not shown to set the roles of users and other actions. So have added the checks for isDialog and isAdmin. Please review the changes and let me know whether I've to make further changes . Thanks!

**Comment by sjkadali** [comment]
@paulbert , thanks for the review. I changed the conditional to check for just !isDialog. This is how the members page for admin looks now. 

Screenshot from 2020-06-23 13-23-39

Is this the intended behaviour?

File: src/app/users/users-table.component.ts [hunk 1]
```diff
@@ -136,6 +136,9 @@ export class UsersTableComponent implements OnInit, OnDestroy, AfterViewInit, On
   }
 
   ngAfterViewInit() {
+    if (!this.isUserAdmin && !this.isDialog) {
```
Comment (paulbert) [inline src/app/users/users-table.component.ts:139]: @sjkadali This line should only check `isDialog`, not `isUserAdmin`.

**Comment by sjkadali** [comment]
@paulbert , have addressed the requested changes. Please review. Thanks1

**Pull request merged**